src/TypographyExample.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { FunctionComponent } from "react"
+import { StyleSheet, View, Text } from "react-native"
+
+import { Sizing, Typography } from "./styles"
+
+const TypographyExample: FunctionComponent = () => {
+  return (
+    <View style={style.container}>
+      <View style={style.headerContainer}>
+        <Text style={style.headerText}>Typography</Text>
+      </View>
+
+      <View style={style.sectionContainer}>
+        <View style={style.subHeaderContainer}>
+          <Text style={style.subHeaderText}>Header</Text>
```

Comment:
Given each list item is self labeling, what do you think about removing the header of the section?